authentication/src/main/java/io/camunda/authentication/CamundaOidcUserService.java

```diff
@@ -89,4 +94,11 @@ public OidcUser loadUser(final OidcUserRequest userRequest) throws OAuth2Authent
                 .map(GroupEntity::name)
                 .toList()));
   }
+
+  private String getUsernameFromClaims(final Map<String, Object> claims) {
+    final var usernameClaimName =
+        securityConfiguration.getAuthentication().getOidc().getUsernameClaim();
+
+    return claims.get(usernameClaimName).toString();
```

Comment:
:wrench: This should probably not result in an NPE but something more helpful ("Username claim with key %s not found, ...")